logger/logger.go

```diff
@@ -6,7 +6,7 @@ import (
 	"time"
 
 	"github.com/jsternberg/zap-logfmt"
-	isatty "github.com/mattn/go-isatty"
+	"github.com/mattn/go-isatty"
```

Comment:
Can this be kept? It's something that goimports adds automatically. The intention is because the package has `isatty` as its name. While it's not necessary to "rename" it, the rename is added to make it explicit that the package's name differs from its basename so it's easier for a human to see, "Oh, go-isatty is what's creating the itatty package name."